zh/star/coremodule/branding/org/netbeans/core/startup/Bundle_star_zh.properties

```diff
@@ -12,14 +12,14 @@ ACSN_AboutTabs=N/A
 ACSD_AboutTabs=N/A
 
 LBL_splash_window_title=\u6b63\u5728\u542f\u52a8STAR-CCM+
-SPLASH_WIDTH=658
-SPLASH_HEIGHT=500
+SPLASH_WIDTH=600
+SPLASH_HEIGHT=300
 
 SplashShowProgressBar=\u5047

```

Comment:
done

On Thu, Aug 25, 2016 at 12:01 PM, pyleeric <notifications@github.com> wrote:

> In zh/star/coremodule/branding/org/netbeans/core/startup/
> Bundle_star_zh.properties
> <https://github.com/cd-adapco/i18n/pull/2#discussion_r76272626>:
>
> > @@ -12,14 +12,14 @@ ACSN_AboutTabs=N/A
> >  ACSD_AboutTabs=N/A
> >
> >  LBL_splash_window_title=\u6b63\u5728\u542f\u52a8STAR-CCM+
> > -SPLASH_WIDTH=658
> > -SPLASH_HEIGHT=500
> > +SPLASH_WIDTH=600
> > +SPLASH_HEIGHT=300
> >
> >  SplashShowProgressBar=\u5047
>
> Note Unicode 5047 represents 'false' in Chinese
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cd-adapco/i18n/pull/2/files/a7d17dbf9d21fb502ee559de3e80a0085b96c678#r76272626>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADVVdnk-hPf_dx54VPnXfSZAc1xvwc5Fks5qjbxfgaJpZM4JtOw6>
> .
>



-- 

*Peter Weatherdon *
Director: Production
Lebanon
603-277-3139 (T)
603-558-1443 (M)
peter.weatherdon@cd-adapco.com  <peter.weatherdon@cd-adapco.com>
http://www.cd-adapco.com